### DISCUSSION TITLE
YouTube Support ?

### DISCUSSION THREAD

**Discussion by ZloyKott** [Chatterino/chatterino2#3140] [Ideas]
Why not add support Youtube like Rutony chat, Sheep Chat, Streamix ?

**Comment by Mm2PL** [comment]
We've discussed this [here](https://github.com/Chatterino/chatterino2/issues/357), [here](https://github.com/Chatterino/chatterino2/issues/2162) and probably more in the discord.

**Reply by ZloyKott** [reply]
ok, but can you help me if i like a noob? )))
If there is an opportunity to do this - help

**Reply by ALazyMeme** [reply]
No, there's far too much work involved with supporting platforms with terrible chat platforms like youtube. If someone wants to implement it, sure. But it won't be a main focus of the development.

**Reply by apa420** [reply]
Implementing a youtube chat would require A LOT of work since it doesn't work on irc like twitch chat does and instead uses websockets or something like that. So basically little code can be reused for that.
